ac1/_sources/projects/module_b.rst

```diff
@@ -13,93 +13,181 @@ Project Description
-        they?
-    -   Did you make any trade-offs as you were cleaning? What and why?
+- **Deliverable:** State and describe your dataset. Why did you choose this
+  particular dataset?
 
-4.  Join the multiple sheets across a common column. You will need to join
-    datasets using vlookup.
+*Hints:*
 
-    -   What did you choose as the joining key? Why?
+- Places to find free datasets: `World Bank Open Data`_, `FiveThirtyEight`_,
+  `Kaggle`_
+- `Additional list of websites to find datasets`_
+- Revisit the importing data section from module B for review on how to import
+  data
+
+
+**Step 2: Clean Your Dataset**
+
+Only rarely will datasets ever be ready for analysis right off the bat.
+Therefore, you will need to prepare your dataset to make useful observations.
+Don’t worry if it seems overwhelming at first. You want to remove outliers that
+may skew your dataset, while still maintaining the integrity of your data. Some
+steps have been provided to help with this process. Once you clean your dataset,
+you should be able to find sections of the data that are interesting and find
+relevant relationships.
+
+**To Do:**
+
+- First look at your data and see if you can spot any inconsistencies
+- Filter out unwanted outliers
+- Check for missing data values
+- **Deliverable:** Summarize your data cleaning process and make sure to
+  answer the following questions.
 
-5.  Using the joined dataset, find summary statistics for the population.
+  - Are there any ethical issues with the way you cleaned the data? What are
+    they?
+  - Did you make any trade-offs as you were cleaning? What were these tradeoffs
+    and why did you make this decision?
+
+*Hints:*
+  - For a more in-depth explanation of data cleaning `read this.`_
 
-    -   For all of the numeric variables, what are the population-wide mean,
-        median, variance, standard deviation?
+**Step 3: Consolidate and Summarize your Data**
 
-6.  Choose subsets of the data that you find interesting. Find summary
-    statistics for the numeric variables, within those subsets of the data.
+After cleaning up your data, you will want an overview of what your data is
+saying. Sometimes when working with a large dataset, your data will be split
+across different sheets, making it difficult to find summary statistics. If
+that is the case, you will need to join common sections first before finding
+interesting statistics about your data.
 
-    -   Explain why you chose this set of groups.
-    -   Within subsets of interest, what are the count, mean, median, variance,
-        standard deviation?
-    -   Is the sample size enough within each group? What does this imply for
-        reliability of summary statistics, and for privacy considerations?
+**To Do:**
 
-7.  Present the grouped summary statistics in the report.
+- Join the multiple sheets across a common column if necessary. You will need to
+  join datasets using vlookup
```

Comment:
Capitalize VLOOKUP

Reply:
Also, end that bullet with a period. All bullets in a list should be formatted the same, so since the others are punctuated, this one should be too.

Reply:
Make sure to check this for all the bullet lists throughout as well.